spec/rubocop/cop/style/safe_navigation_spec.rb

```diff
@@ -0,0 +1,414 @@
+                      '  puts 1',
+                      'end']
+
+            new_source = autocorrect_source(cop, source)
+
+            expect(new_source).to eq(['if foo&.bar(baz)',
+                                      '  puts 1',
+                                      'end'].join("\n"))
+          end
+
+          it 'registers an offense for a check using respond_to before ' \
+             'a method call with block' do
+            source = ['if foo.respond_to?(:bar) && foo.bar { |e| e.call }',
+                      '  puts 1',
+                      'end']
+
+            new_source = autocorrect_source(cop, source)
+
+            expect(new_source).to eq(['if foo&.bar { |e| e.call }',
+                                      '  puts 1',
+                                      'end'].join("\n"))
+          end
+
+          it 'registers an offense for a check using respond_to before ' \
+             'a method call with parameters and a block' do
+            source = ['if foo.respond_to?(:bar) && foo.bar(baz) { |e| e.call }',
+                      '  puts 1',
+                      'end']
+
+            new_source = autocorrect_source(cop, source)
+
+            expect(new_source).to eq(['if foo&.bar(baz) { |e| e.call }',
+                                      '  puts 1',
+                                      'end'].join("\n"))
+          end
+
+          it 'registers an offense for a check using respond_to before ' \
+             'a method call' do
+            source = 'puts 1 if foo.respond_to?(:bar) && foo.bar'
+
+            new_source = autocorrect_source(cop, source)
+
+            expect(new_source).to eq('puts 1 if foo&.bar')
+          end
+
+          it 'registers an offense for a check using respond_to before ' \
+             'a method call with an argument' do
+            source = 'puts 1 if foo.respond_to?(:bar) && foo.bar(baz)'
+
+            new_source = autocorrect_source(cop, source)
+
+            expect(new_source).to eq('puts 1 if foo&.bar(baz)')
+          end
+
+          it 'registers an offense for a check using respond_to before ' \
+             'a method call with a block' do
+            source = 'puts 1 if foo.respond_to?(:bar) && foo.bar { |e| e.baz }'
+
+            new_source = autocorrect_source(cop, source)
+
+            expect(new_source).to eq('puts 1 if foo&.bar { |e| e.baz }')
+          end
+
+          it 'registers an offense for a check using respond_to before ' \
+             'a method call with arguments and a block' do
+            source =
+              'puts 1 if foo.respond_to?(:bar) && foo.bar(baz) { |e| e.baz }'
+            new_source = autocorrect_source(cop, source)
+
+            expect(new_source).to eq('puts 1 if foo&.bar(baz) { |e| e.baz }')
+          end
+        end
+      end
+    end
+  end
+
+  context 'target_ruby_version < 2.3', :ruby19 do
+    it 'registers an offense for method calls safe guarded by a ' \
```

Comment:
I think this should be "it does not register an offense ..." or "it registers no offense ..."